src/usecases/retrieveHospitalWardVisitTotals.js

```diff
@@ -0,0 +1,44 @@
+export default ({ getDb }) => async (hospitalId) => {
+  const db = await getDb();
+
+  const queryResult = await db.any(
+    `SELECT
+      wards.id as ward_id,
+      wards.name as ward_name,
+      SUM(totals.total) AS total_visits
+    FROM ward_visit_totals AS totals
+      JOIN wards ON wards.id = totals.ward_id
+    WHERE wards.hospital_id = $1
+    GROUP BY wards.id`,
+    [hospitalId]
+  );
+
+  let wards = [];
+
+  queryResult.map(({ ward_id, total_visits }) => {
+    wards[ward_id] = parseInt(total_visits);
+  });
+
+  const sortedResults = queryResult.sort(
+    (a, b) => b.total_visits - a.total_visits
+  );
+
+  const mostVisitedResult = sortedResults[0];
```

Comment:
```suggestion
  const mostVisitedWard = sortedResults[0];
```

Or `wardWithMostVisits`?